pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.ruff]
+target-version = "py39"
```

Comment:
Is this correct? CI workflow uses Python 3.12.

Reply:
Updated to use Python 3.12.